File: android/src/androidTest/java/org/ligi/passandroid/TestComponent.java
package org.ligi.passandroid;

import dagger.Component;
import javax.inject.Singleton;

@Singleton
@Component(modules = { TestModule.class })
public interface TestComponent extends AppComponent {
    void inject(TheFullscreenBarcodeActivity theFullscreenBarcodeActivity);

    void inject(ThePassEditActivity thePassEditActivity);

    void inject(ThePassViewActivity thePassViewActivity);

    void inject(ThePastLocationsStore thePastLocationsStore);

    void inject(TheBarCodeEditing theBarCodeEditing);

    void inject(ThePassListSwiping thePassListSwiping);

    void inject(TheFieldListEditFragment theFieldListEditFragment);
}

File: android/src/androidTest/java/org/ligi/passandroid/TestComponent.kt
package org.ligi.passandroid

import dagger.Component
import javax.inject.Singleton

@Singleton
@Component(modules = arrayOf(TestModule::class))
interface TestComponent : AppComponent {

    fun inject(theFullscreenBarcodeActivity: TheFullscreenBarcodeActivity)

    fun inject(thePassEditActivity: ThePassEditActivity)

    fun inject(thePassViewActivity: ThePassViewActivity)

    fun inject(thePastLocationsStore: ThePastLocationsStore)

    fun inject(theBarCodeEditing: TheBarCodeEditing)

    fun inject(thePassListSwiping: ThePassListSwiping)

    fun inject(theFieldListEditFragment: TheFieldListEditFragment)
}
